src/utils/crypto.ts

```diff
@@ -167,39 +181,43 @@ export function decryptKey (password, encrypted) {
 
 /**
  * Generate signature
- * @rtype (data: String|Buffer, privateKey: Buffer) => Buffer
- * @param {String|Buffer} data - Data to sign
- * @param {String|Buffer} privateKey - Key to sign with
- * @return {Buffer|Uint8Array} Signature
+ * @rtype (data: String | Buffer, privateKey: Buffer) => Buffer
+ * @param {String | Buffer} data - Data to sign
+ * @param {String | Buffer} privateKey - Key to sign with
+ * @return {Buffer | Uint8Array} Signature
  */
-export function sign (data, privateKey) {
+export function sign (
+  data: string | Buffer, privateKey: string | Buffer): Buffer | Uint8Array {
```

Comment:
```suggestion
  data: string | Buffer, privateKey: string | Buffer): Uint8Array {
```
As I see, `nacl` doesn't aware of `Buffer`, so all it's methods return `Uint8Array`s in all environments